packages/icons/src/companies/index.js

```diff
@@ -20,7 +20,14 @@ const companyLookup = {
 };
 
 function getCompanyIcon(name) {
-  const icon = companyLookup[name.toLowerCase()];
+  const icon =
+    companyLookup[name.toLowerCase()] ||
+    /*
+    Some company names are lyft_pdx instead of just Lyft. 
+    This PR matches based on the first 4 characters to hopefully 
+    match more company icons.
+    */
+    companyLookup[name.toLowerCase().slice(4)];
```

Comment:
wouldn't we want to do `.slice(0,4)` instead to extract "lyft" from "lyft_pdx" or do we want "_pdx"

Reply:
Amazing catch thank you!